utils/config.py

```diff
@@ -30,6 +30,9 @@ def network_name() -> Optional[str]:
 elif network_name() in ("holesky", "holesky-fork"):
     print(f'Using {color("cyan")}config_holesky.py{color} addresses')
     from configs.config_holesky import *
+elif network_name() in ("hoodi-devnet", "hoodi-devnet-remote"):
```

Comment:
Do you need to add these networks to **network-config.yaml**?

Reply:
This piece of code was needed for devnet, now it's not needed, I'll delete it.